yjit/src/codegen.rs

```diff
@@ -5143,8 +5176,14 @@ fn gen_send_iseq(
     // Number of locals that are not parameters
     let num_locals = unsafe { get_iseq_body_local_table_size(iseq) as i32 } - (num_params as i32);
 
-    // Check for interrupts
-    gen_check_ints(asm, side_exit);
+    // Inline ISEQ if possible. Captured operands are not supported yet, which require ep.
+    let inline_method = captured_opnd.is_none() && can_inline_iseq(iseq);
+    incr_counter!(inlined_block_count);
```

Comment:
I'm a bit confused. This looks like it's incrementing the counter here regardless of the value of inline_method. Is that intended?

Reply:
Oh thanks, you're right. I should fix it and get the numbers again.